spec/unit/gateway/contentful_gateway_spec.rb

```diff
@@ -178,4 +178,38 @@
       ])
     end
   end
+
+  context '#When no case study is found' do
+    let(:space_id) { 'owl' }
+    let(:access_token) { 'hoot' }
+    let(:slug) { 'hoot-primary-school-case-study' }
+    let(:contentful_gateway) do
+      described_class.new(space_id: space_id, access_token: access_token)
+    end
+    let(:initial_url) do
+      "https://cdn.contentful.com/spaces/#{space_id}/environments/master/content_types?limit=1000"
+    end
+    let(:case_study_url) do
+      "https://cdn.contentful.com/spaces/#{space_id}/environments/master/entries?content_type=caseStudy&include=10&fields.slug=#{slug}"
+    end
+    let(:case_study) { contentful_gateway.get_case_study(slug: slug) }
+
+    before do
+      headers['Authorization'] = "Bearer #{access_token}"
+
+      stub_request(:get, initial_url)
+        .with(headers: headers)
+        .to_return(status: 200, body: response_with_no_items, headers: {})
+
+      stub_request(:get, case_study_url)
+        .with(headers: headers)
+        .to_return(status: 200, body: response_with_no_items, headers: {})
+
+      case_study
+    end
+
+    it 'returns nil' do
```

Comment:
```suggestion
    it 'can return nil when a Contentful case study is not found ' do
```

Similar reason as above, so that it aligns with the description of the other tests.

Reply:
👍 Updated in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/9/commits/f6a135520d6e96f640b524d4e065244a29a3cc3c